dotCMS/src/integration-test/java/com/dotcms/content/elasticsearch/business/ESMappingAPITest.java

```diff
@@ -572,4 +573,83 @@ public void testLoadCategories_GivenContentWithCats_ShouldLoadESMapWithListOfCat
 
     }
 
+    /**
+     * General purpose is testing that KeyValue fields are indexed correctly.
+     * Given scenario: We create a Content type that holds a Key value field.
+     * Expected Results: Then we feed data into such field and use an ES query over the KeyValue to verify the results are coming back.
+     * @throws DotDataException
+     * @throws DotSecurityException
+     */
+    @Test
+    public void Test_Create_ContentType_With_KeyValue_Field_Test_Query_Expect_Success()
+            throws DotDataException, DotSecurityException {
+
+        List<Field> fields = new ArrayList<>();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'fields' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5533307)